docs/guides/connecting_to_your_data/datasource_configuration/sql_components/_tab_data_connector_example_configurations_runtime.mdx

```diff
@@ -9,21 +9,7 @@ import TipCustomDataConnectorModuleName from '../components/_tip_custom_data_con
 
 At this point, your configuration should look like:
 
-```python
-datasource_config: dict = {
-    "name": "my_datasource_name",
-    "class_name": "Datasource",
-    "module_name": "great_expectations.datasource",
-    "execution_engine": {
-        "class_name": "SqlAlchemyExecutionEngine",
-        "module_name": "great_expectations.execution_engine",
-        "connection_string": CONNECTION_STRING,
-    },
-    "data_connectors": {
-        "name_of_my_runtime_data_connector": {}
-        }
-    }
-}
+```python name="sql datasource configuration with empty configured sql data_connector"
```

Comment:
These blocks are all referencing a ConfiguredAssetSqlDataConnector rather than a RuntimeDataConnector

Reply:
Should be resolved now.